projects/uitdatabank/docs/entry-api/events/create-uitpas.md

```diff
@@ -1,10 +1,275 @@
-Coming soon.
-
-<!-- 
-  @todo
-  Permissions: Who can create UiTPAS events?
-  Refer to the guide about creating new events first.
-  Explain what UiTPAS events are, and the extra required fields.
-  Also link to https://docs.publiq.be/docs/uitpas/6e03991383b32-registering-events and vice-versa. (Move everything under the "Entry API" header here and make any necessary adjustments/additions, then link on that page to this one.)
-  Mention that the audience should be set to "members" if the UiTPAS event should not be visible on online calendars like https://www.uitinvlaanderen.be
--->
+# UiTPAS events
+
+An UiTPAS event is a regular event that has ticketing options with an [UiTPAS](https://docs.publiq.be/docs/uitpas/introduction) discount.
+
+Event organizers that offer UiTPAS discounts can get (partial) reimbursements for those discounts by [registering their ticket sales in UiTPAS](https://docs.publiq.be/docs/uitpas/ticket-sales/registering). However, ticket sales can only be registered for known UiTPAS events.
+
+To register your UiTPAS event first, you need to register it in UiTdatabank. This way it will also appear on online calendars like [UiTinVlaanderen](https://www.uitinvlaanderen.be). After registering it in UiTdatabank, it will automatically also be registered in UiTPAS.
+
+In this guide you will learn how to:
+
+* Create an UiTPAS event in UiTdatabank
+* Get the UiTPAS prices for an event in UiTdatabank
+* Configure which card systems (for example UiTPAS Gent, UiTPAS Oostende, ...) should provide discounts for their passholders when they buy tickets for your event
+* Change an UiTPAS event to a regular event instead
+* Hide an UiTPAS event from online calendars
+
+Before diving in, make sure you have read the following guides first:
+
+* [Creating a new event](./create.md), as creating an UiTPAS event builds upon that.
+* [What are organizers?](../organizers/introduction.md) to learn about the concept of organizers and more specifically, UiTPAS organizers.
+* The guide about [linking an organizer to an event](../organizers/linking.md), which is required for UiTPAS events.
+* The guide about [adding price info to your event](../shared/price-info.md), which is required for UiTPAS events.
+
+## Required permissions
+
+As with regular events, anyone can create new UiTPAS events in UiTdatabank by using either a user access token or a client access token.
+
+The user or client that created the event will become the `creator` of the online event, which allows them to later make changes to it or delete it.
+
```

Comment:
Shouldn't you better explicitize that other users than the creator by default don't have write access to these events? (and that they won't appear in the UDB overview on the front page)
(just a thought, not blocking)

Reply:
Good point, I will add that (and also to other "creating an ... event" guides)

Reply:
For now I've added an extra sentence that refers to the update guide for more info about permissions required to make updates. That felt like the best place to go into more detail about who can(not) make updates to existing events.